deepchem/feat/tests/test_roberta_tokenizer.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+from deepchem.feat.roberta_tokenizer import RobertaFeaturizer
```

Comment:
We should have a try/catch block here

Reply:
I'd say something like:

```
try:
  from deepchem.feat.roberta_tokenizer import RobertaFeaturizer
except ModuleNotFoundError:
  pass

Reply:
Should we not log/warn for this essentially breaking lack of dependency?

Reply:
Actually now that I think about it, this should just be a local import within the tests! The issue is that this is a global import at the top of the file rather than a local import that's secured by the the pytest mark guard